aten/src/ATen/native/quantized/cpu/conv_serialization.h

```diff
@@ -123,8 +123,6 @@ ConvParamsSerializationTypeV3 parse_conv_serialized_state(c10::IValue v) {
     torch::List<at::Tensor> dilation_x_kSpatialDim = elements[4].toTensorList();
     at::Tensor groups = elements[5].toTensor();
 
-    std::vector<c10::optional<at::Tensor>> optional;
-
     std::vector<int64_t> config_vals;
```

Comment:
Hmm, seems like config_vales should have .reserve() called on it with the various of sizes of stuff being added to it.

Reply:
It is hard to reserve() in some cases

Reply:
Should just be
`.reserve(stride_x_kSpatialDim.size() + padding_x_kSpatialDim.size() + dilation_x_kSpatialDim.size() + kSpatialDim + 3)`
right?

Reply:
Thanks for pointing it out, I have added.